notebooks/learning-to-rank/01-learning-to-rank.ipynb

```diff
@@ -0,0 +1,1556 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TKxL_NmZmOpF"
+      },
+      "source": [
+        "# How to train and deploy Learning To Rank\n",
+        "\n",
+        "TODO: udpate the link to elastic/elasticsearch-labs instead of my fork before merging.\n",
+        "\n",
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/afoucret/elasticsearch-labs/blob/ltr-notebook/notebooks/learning-to-rank/01-learning-to-rank.ipynb)\n",
```

Comment:
Don't forget to change this before merging 😉

Reply:
Done.